parsers/src/test/scala/uk/gov/ons/addressIndex/parsers/PostCodeValidationTest.scala

```diff
@@ -226,56 +98,68 @@ class PostCodeValidationTest extends FlatSpec with Matchers {
   }
 
   it should "RH1 -> RH1" in {
-    testHelper(
-      input = "RH1",
-      expected = "RH1"
+    // Given
+    val input = Map(
+      Tokens.postcode -> "RH1"
+    )
+
+    val expected = Map(
+      Tokens.postcode -> "RH1",
+      Tokens.postcodeIn -> "RH1"
     )
+
+    // When
+    val actual = Tokens.postTokenizeTreatmentPostCode(input)
+
+    // Then
+    actual shouldBe expected
   }
 
   it should "L1 -> L1" in {
-    testHelper(
-      input = "L1",
-      expected = "L1"
+    // Given
+    val input = Map(
+      Tokens.postcode -> "L1"
     )
+
+    val expected = Map(
+      Tokens.postcode -> "L1",
+      Tokens.postcodeIn -> "L1"
+    )
+
+    // When
+    val actual = Tokens.postTokenizeTreatmentPostCode(input)
+
+    // Then
+    actual shouldBe expected
   }
 
   it should "1AA -> 1AA" in {
-    testHelper(
-      input = "1AA",
-      expected = "1AA"
+    // Given
+    val input = Map(
+      Tokens.postcode -> "1AA"
     )
+
+    val expected = Map(
+      Tokens.postcode -> "1AA",
+      Tokens.postcodeIn -> "1AA"
```

Comment:
this is actually postcodeIn, but to be consistent, should probably be postcodeOut